crates/re_viewer/build.rs

```diff
@@ -1,4 +1,212 @@
+use std::path::Path;
```

Comment:
it would be helpful with a `//!` comment here explaining in short the purpose of this build-script